cloud/openstack/os_facts.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+
+# Copyright (c) 2014 Hewlett-Packard Development Company, L.P.
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+try:
+    import shade
+    from shade import meta
+    HAS_SHADE = True
+except ImportError:
+    HAS_SHADE = False
+
+DOCUMENTATION = '''
+---
+module: os_server_facts
+short_description: Retrieve facts about a compute instance
+extends_documentation_fragment: openstack
+description:
+   - Retrieve facts about a server instance from OpenStack.
+notes:
+   - Facts are placed in the C(openstack) variable.
+options:
+   server:
+     description:
+        - Name or ID of the instance
+     required: true
+   mounts:
+     description:
+        - Optional list of dicts tying volumes to mount points
```

Comment:
Thanks for the catch - this should actually be removed for now - it's left over from a previous attempt to get and end to end flow of "create server, create volume, mount volume, install software" - but since cinder volumes don't have a way to express what mount point they were created for, it gets awkward. However, this can be added later if it turns out it's actually needed.